pkg/subctl/operator/submarinerop/crds/ensure.go

```diff
@@ -40,7 +45,12 @@ func Ensure(restConfig *rest.Config) (bool, error) {
 	// Attempt to update or create the CRD definition
 	// TODO(majopela): In the future we may want to report when we have updated the existing
 	//                 CRD definition with new versions
-	return utils.CreateOrUpdateCRD(crdUpdater, submarinerCrd)
+	submarinerCreated, err := utils.CreateOrUpdateCRD(crdUpdater, submarinerCrd)
```

Comment:
You can use 
```
 CreateOrUpdateEmbeddedCRD(updater crdutils.CRDUpdater, crdYaml string) (bool, error
```
In this case, and save the getBrokerCRD / getSubmarinerCRD :)

Reply:
Indeed, thanks!